src/lib/base64.c

```diff
@@ -45,36 +45,41 @@ static const unsigned char b64dec[256] = {
 
 void base64_encode(const void *src, size_t src_size, buffer_t *dest)
 {
+	const size_t res_size = MAX_BASE64_ENCODED_SIZE(src_size);
+	unsigned char *beg = buffer_append_space_unsafe(dest, res_size);
```

Comment:
"beg" isn't much shorter than "begin" or "start".  Since those are IMO clearer, I'll change this when I commit it. Thanks!

Reply:
Sure! Personally I prefer short names with equal length such as 'beg', 'end', 'cur', 'ptr', etc